src/utils/Dom.ts

```diff
@@ -19,6 +19,15 @@ export interface IOffset {
 export class Dom {
   private static CLASS_NAME_REGEX = /-?[_a-zA-Z]+[_a-zA-Z0-9-]*/g;
   private static ONLY_WHITE_SPACE_REGEX = /^\s*$/;
+  private static forceNativeEvent = false;
+
+  // Force the use of native javascript event.
+  public static native(value?: boolean) {
```

Comment:
Why is that optional, and why is there a type check on "boolean" ?

Reply:
I think i will change that for a simple static attribute.

Reply:
Is there a reason why not to make the property `public` and remove this method? Can the documentation be added to the property?

If not, then I have the same comment as above for the name of the method. If I called `Dom.native()`, I would be guessing what that means. Improving the name would remove any doubt.